assets/railcraft/lang/zh_CN.lang

```diff
@@ -1230,3 +1163,92 @@ tile.railcraft.wall.stone.brick.name=石砖墙
+track_kit.railcraft.railcraft_activator.tip=激活经过的矿车\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_booster.name=推进
+track_kit.railcraft.railcraft_booster.tip=为经过的矿车加力\n若未激活则会为矿车减速\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_buffer.name=缓停
+track_kit.railcraft.railcraft_buffer.tip=用于轨道尽头\n§o§9- 通过撬棍敲击来变向 -
+
+track_kit.railcraft.railcraft_control.name=控制
+track_kit.railcraft.railcraft_control.tip=提供轻微的多向加速.\n§o§9- 通过撬棍敲击来变向 -\n§o§c- 通入红石信号亦可变向 -
+
+track_kit.railcraft.railcraft_coupler.name=耦合
+track_kit.railcraft.railcraft_coupler.tip=将经过的车厢连接/分离\n§o§9- 通过撬棍敲击来切换模式 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_detector.name=感应
+track_kit.railcraft.railcraft_detector.tip=车辆经过时发出红石信号.
+
+track_kit.railcraft.railcraft_detector.travel.name=方向感应
+track_kit.railcraft.railcraft_detector.travel.tip=车辆经过指定方向时\n发出红石信号.\n§o§9- 通过撬棍敲击来变向 -
+
+track_kit.railcraft.railcraft_disembarking.name=驻车
+track_kit.railcraft.railcraft_disembarking.tip=按箭头方向放下乘客.\n§o§9- 通过撬棍敲击来变向 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_dumping.name=倾倒
+track_kit.railcraft.railcraft_dumping.tip=悬空放置\n倾倒出乘坐实体\n§o§c- 使用红石信号禁用 -
+
+track_kit.railcraft.railcraft_embarking.name=乘车
+track_kit.railcraft.railcraft_embarking.tip=使附近实体乘上车厢\n§o§9- 通过撬棍敲击来调整范围 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_gated.name=门控
+track_kit.railcraft.railcraft_gated.tip=内建门控装置的轨道\n§o§c- 使用红石信号开关门 -
+
+track_kit.railcraft.railcraft_gated.one.way.name=单向门控
+track_kit.railcraft.railcraft_gated.one.way.tip=内建单向门控装置的轨道\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_launcher.name=发射
+track_kit.railcraft.railcraft_launcher.tip=让矿车一飞冲天~\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_limiter.name=机车限速
+track_kit.railcraft.railcraft_limiter.tip=调整经过机车的速度\n§o§9- 通过撬棍敲击来切换模式 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_locking.name=封锁
+track_kit.railcraft.railcraft_locking.tip=发动或驻留矿车\n§o§9- 通过撬棍敲击来切换模式 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_locomotive.name=机车调整
+track_kit.railcraft.railcraft_locomotive.tip=发动或停止机车\n§o§9- 通过撬棍敲击来切换模式 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_missing.name=???
+track_kit.railcraft.railcraft_missing.tip=只是拿来占位的，这说明这个轨道出问题了
+
+track_kit.railcraft.railcraft_one.way.name=单向
+track_kit.railcraft.railcraft_one.way.tip=只允许矿车通向箭头标定方向\n§o§9- 通过撬棍敲击来变向 -
+
+track_kit.railcraft.railcraft_priming.name=起爆
+track_kit.railcraft.railcraft_priming.tip=引爆TNT矿车,/n可调引信长度\n§o§9- 通过撬棍敲击来设定引信长度 -\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_routing.name=路由
+track_kit.railcraft.railcraft_routing.tip=设置经过机车的车票\n§o§9- 通过撬棍敲击来改变车票 -
+
+track_kit.railcraft.railcraft_transition.name=高速过渡
+track_kit.railcraft.railcraft_transition.tip=用于高速与正常速度的过渡\n§o§c- 使用红石信号激活 -
+
+track_kit.railcraft.railcraft_whistle.name=鸣笛
+track_kit.railcraft.railcraft_whistle.tip=经过的机车会鸣笛\n§o§c- 使用红石信号激活 -
+
+track_type.railcraft.railcraft_iron.name=铁制
+track_type.railcraft.railcraft_strap_iron.name=镀铁
+track_type.railcraft.railcraft_abandoned.name=废弃的
```

Comment:
afaik, there isn't a single popular translation for this

On Sun, Aug 28, 2016 at 11:52 PM, Urey. Xue <notifications@github.com>
wrote:

> In assets/railcraft/lang/zh_CN.lang
> <https://github.com/CovertJaguar/Railcraft-Localization/pull/314#discussion_r76534321>
> :
>
> > +
> > +track_kit.railcraft.railcraft_priming.name=起爆
> > +track_kit.railcraft.railcraft_priming.tip=引爆TNT矿车,/n可调引信长度\n§o§9- 通过撬棍敲击来设定引信长度 -\n§o§c- 使用红石信号激活 -
> > +
> > +track_kit.railcraft.railcraft_routing.name=路由
> > +track_kit.railcraft.railcraft_routing.tip=设置经过机车的车票\n§o§9- 通过撬棍敲击来改变车票 -
> > +
> > +track_kit.railcraft.railcraft_transition.name=高速过渡
> > +track_kit.railcraft.railcraft_transition.tip=用于高速与正常速度的过渡\n§o§c- 使用红石信号激活 -
> > +
> > +track_kit.railcraft.railcraft_whistle.name=鸣笛
> > +track_kit.railcraft.railcraft_whistle.tip=经过的机车会鸣笛\n§o§c- 使用红石信号激活 -
> > +
> > +track_type.railcraft.railcraft_iron.name=铁制
> > +track_type.railcraft.railcraft_strap_iron.name=镀铁
> > +track_type.railcraft.railcraft_abandoned.name=废弃的
>
> ...Awaiting itemGroup.railcraft.general
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CovertJaguar/Railcraft-Localization/pull/314/files/6a17692d07b02d372c61d343e3d3111ae8fc0c0e#r76534321>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AE_Kufy_fTjmlpme79CLUvEXK00osIxrks5qka64gaJpZM4Ju8y3>
> .
>